app/main/lib/similarity.py

```diff
@@ -93,6 +104,10 @@ def add_item(item, similarity_type):
   app.logger.info(f"[Alegre Similarity] [Item {item}, Similarity type: {similarity_type}] response for add was {response}")
   return response
 
+def callback_add_item(item, similarity_type):
+  if similarity_type == "audio":
+      return audio_model().add(item)
```

Comment:
probably nothing is listening to this response, should we log it?

Reply:
Fair enough!